crates/rolldown/tests/esbuild/default/warnings_inside_node_modules/artifacts.snap

```diff
@@ -4,135 +4,9 @@ snapshot_kind: text
 ---
 # Errors
 
-## RESOLVE_ERROR
+## UNRESOLVED_ENTRY
```

Comment:
Due to a bug of `integration_test.rs`, the snapshot is actually put into https://github.com/rolldown/rolldown/blob/4e24977b9b2291fc6de1703fe551bc9dfbc70375/crates/rolldown_testing/src/artifacts.snap.

This PR fix the location bug. The snapshot output is the same as old one.

The result isn't expected, but it's not caused by this PR.

Reply:
will you fixed in next pr?

Reply:
Probably in another pr. I need to investigate it a little bit.

Reply:
not hurry if you don't have enough time, since it is not regressed the tests.